### PR TITLE
[[Documentation]] Fixes to replaceText.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
* Removed unneeded html entities.
* Improved examples.
* Added missing descriptions for params.
* Added tag.

### CODE REVIEW

File: docs/dictionary/function/replaceText.lcdoc [hunk 1]
```diff
@@ -60,13 +58,12 @@ the <matchExpression> to make the match case-insensitive.)
 >*Note:* A number of characters in regular expressions have special
 > meanings and need to be escaped with back slashes For example period
 > (".") matches any character, so in order to replace period characters
-> with a regular expression use "\." . For more information on regular
+> with a <regular expression> use "\." . For more information on regular
 > expressions see the Perl documentation here:
-> &lt;http://perldoc.perl.org/perlre.html&gt; 
-
-&nbsp;
+> <http://perldoc.perl.org/perlre.html>
```
Comment (livecodeali) [inline docs/dictionary/function/replaceText.lcdoc]: @asayd URLs don't need to be in angle brackets, they automatically render as links. You can also use `[link text](url)`.

**Comment by asayd** [comment]
Thanks, @livecodeali . I couldn't quite remember the convention.

**Comment by peter-b** [comment]
@livecode-vulcan review ok 5442342

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 5442342093c2844bc439659ac743462d0302d85c

**Comment by livecode-vulcan** [comment]
😎 test success 5442342093c2844bc439659ac743462d0302d85c

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/643)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/648)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/643)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/647)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/652)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/644)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/644)

**Pull request merged**